.releaserc.js

```diff
@@ -67,6 +101,10 @@ module.exports = {
         pkgRoot: './packages/mongodb-memory-server-global-4.4',
       },
     ],
+    ["@semantic-release/git", {
+      "assets": ["packages/*/package.json", "CHANGELOG.md"],
+      "message": "chore: release v${nextRelease.version}"
```

Comment:
should this maybe be changed to `release: v0.0.0`? (adding a new commit-type)

Reply:
Yep `chore` is quite a strange thing for the release commit. I like `release: v0.0.0`